cmd/diff.go

```diff
@@ -36,22 +38,14 @@ func init() {
 }
 
 var diffCmd = &cobra.Command{
-	Use:   "diff [env-name] [-f <file-or-dir>]",
-	Short: "Display differences between server and local config",
+	Use:   "diff [env-name] [env-name] [-f <file-or-dir>]",
```

Comment:
Use here is probably: `diff <env1> [env2] [-f <file-or-dir>]`. Not sure how informative that is without the examples though.

Reply:
Going to follow the `git diff` schematics and do `diff [<env1> [<env2>]] [-f <file-or-dir>] to also capture this case: `ks diff -f ./pod.yaml`